packages/ios_platform_images/example/ios/RunnerTests/IosPlatformImagesTests.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import ios_platform_images
+
+class IosPlatformImagesTests: XCTestCase {
+
+  var mockChannel: MockMethodChannel!
+
+  override func setUp() {
+    super.setUp()
+    mockChannel = MockMethodChannel()
+  }
+
+  func testHandleMethodCall_loadImage() {
+    let name = "flutter"
+
+    let call = FlutterMethodCall(methodName: "loadImage", arguments: [name])
+
+    let mockChannel = MockMethodChannel()
+
+    let plugin = IosPlatformImagesPlugin(channel: mockChannel)
+
+    let resultExpectation = expectation(description: "result block must be called.")
+
+    plugin.handle(call) { result in
+      let result = result as? [String: Any]
+      XCTAssertNotNil(result)
+      
+      let scale = result?["scale"] as? CGFloat
+      let data = result?["data"] as? FlutterStandardTypedData
+
+      XCTAssertNotNil(scale)
+      XCTAssertNotNil(data)
+
+      resultExpectation.fulfill()
+    }
+
+    waitForExpectations(timeout: 2, handler: nil)
```

Comment:
Flutter style prefers not using timeouts in tests, as they contribute to flake. This should just wait until the expectation is fulfilled.

Same in all of the tests below.